service/src/main/resources/db/migration/V281__message_thread_folder.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE message_thread_folder
+(
+    id                      uuid PRIMARY KEY         NOT NULL DEFAULT ext.uuid_generate_v1mc(),
+    created                 timestamp with time zone NOT NULL DEFAULT now(),
+    updated                 timestamp with time zone NOT NULL DEFAULT now(),
+    name                    text                     NOT NULL DEFAULT '',
+    owner_id                uuid                     NOT NULL REFERENCES message_account (id) ON DELETE CASCADE,
+
+    CONSTRAINT uniq$thread_folder UNIQUE (name, owner_id)
+);
+
+CREATE INDEX idx$thread_folder_owner ON message_thread_folder (id, owner_id);
```

Comment:
This index seems useless and should be removed. `id` implicitly has an index because it's the primary key, so this would only be marginally useful for queries that return `id`, `owner_id` pairs.